pybandits/strategy.py

```diff
@@ -73,32 +118,44 @@ class BestActionIdentification(Strategy):
 
     Parameters
     ----------
-    exploit_p: Float_0_1 (default=0.5)
+    exploit_p: Optional[Float01], 0.5 if not specified
```

Comment:
Let's call this class BestActionIdentificationBandit to keep it homogeneous with the others